examples/games/breakout.rs

```diff
@@ -218,27 +218,23 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
 
     // Scoreboard
     commands.spawn_bundle(TextBundle {
-        text: Text {
-            sections: vec![
-                TextSection {
-                    value: "Score: ".to_string(),
-                    style: TextStyle {
-                        font: asset_server.load("fonts/FiraSans-Bold.ttf"),
-                        font_size: SCOREBOARD_FONT_SIZE,
-                        color: TEXT_COLOR,
-                    },
-                },
-                TextSection {
-                    value: "".to_string(),
-                    style: TextStyle {
-                        font: asset_server.load("fonts/FiraMono-Medium.ttf"),
-                        font_size: SCOREBOARD_FONT_SIZE,
-                        color: SCORE_COLOR,
-                    },
-                },
-            ],
-            ..default()
-        },
+        text: Text::with_section(
```

Comment:
It is a little awkward to construct multi-section text this way, as the text alignment gets written between the first section and the second (it is more obvious if the text alignment is written out as a literal). Maybe we should add a `new_empty_with_alignment` method to allow constructing no-section text and inserting sections one by one?

Reply:
I'm open to this sort of API improvement, but it belongs in a separate PR.